### ISSUE TITLE
Quality for jpges not changing

### ISSUE THREAD

**Issue by kgdizains** [oscarotero/imagecow#44]
Hi, i have this code:
`      $quality = 80;
        $image = Image::fromFile($file);
        $image->autoRotate();
        if(!empty($wm)){
            $watermark = Image::fromFile($wm);
            $image->watermark($watermark, $x = 'right', $y = 'bottom');
        }
        $image->quality($quality);
        $image->resize($w, $h);
        $image->save($dest);`

I can set quality 10 or 90 the image size (in kb) and quality afters save is the same :(

**Comment by oscarotero** [comment]
What library are you using? Gd or Imagick?

**Comment by kgdizains** [comment]
Im not defined library, but i think by default is GD :)

**Comment by oscarotero** [comment]
I've reproduced your code here https://github.com/oscarotero/imagecow/blob/master/example/quality.php
and it works fine to me.

**Comment by kasparssilins** [comment]
Ok, i now get it where is problem...
When you change quality and $image -> show(); <--- Yes, quality changed!
but here: $image -> save('low-ress-image.jpg'); <--- Quality not changing..
I hope you understand ;)

**Comment by oscarotero** [comment]
To me, both methods works fine. "low-less-image.jpg" has the same low quality than show()

**Comment by kasparssilins** [comment]
To mo save() function not change image quality :(
PHP 7.3, Apache, GD

**Comment by kasparssilins** [comment]
here is demo: https://jek-siltums.the-webapps.com/imagecow/tests.php

**Comment by oscarotero** [comment]
No idea. This code works fine to me (save and show). Are you sure that the new image is really saved in this directory and is the same that is loaded by the html?

**Comment by kasparssilins** [comment]
Yes, it's generate new image every time when this url is opened!
This is standard shared hosting, later i will try on my own local server, let's see how this code will work there!
Thanks for your time and response!

**Comment by lkunxyz** [comment]
try this tool [jpeg](https://jpegtojpg.com/zh)
